commit 704beb265cd6f860a10f4a5ae0fd8c52e72bf392
Author: Andrew Price <anprice@redhat.com>
Date:   Tue Jul 19 15:41:13 2016 +0100

    gfs2(5): Clarify the availability of the loccookie option
    
    The loccookie option was added in kernel-3.10.0-341.el7 but the first
    release will be RHEL 7.3 so use that to clarify when 'loccookie' appears
    in RHEL7.
    
    Resolves: rhbz#1326508
    
    Signed-off-by: Andrew Price <anprice@redhat.com>

diff --git a/gfs2/man/gfs2.5 b/gfs2/man/gfs2.5
index 56d1a00..6c77519 100644
--- a/gfs2/man/gfs2.5
+++ b/gfs2/man/gfs2.5
@@ -193,7 +193,7 @@ give out well over half a billion location based cookies. This option was added
 in the 4.5 Linux kernel. Prior to this kernel, gfs2 did not add directory
 entries in a way that allowed it to use location based readdir cookies.
 \fBNote:\fP To safely turn on this option, all nodes mounting the filesystem
-must be running at least a 4.5 Linux kernel. If this option is only enabled on
+must be running at least a 4.5 Linux kernel or RHEL 7.3. If this option is only enabled on
 some of the nodes mounting a filesystem, the cookies returned by nodes using
 this option will not be valid on nodes that are not using this option, and vice
 versa.  Finally, when first enabling this option on a filesystem that had been
